### ISSUE TITLE
Unable to log in

### ISSUE THREAD

**Issue by johnhaley81** [ajrosen/Alfred#4]
Thank you for the workflow! I'm very excited to try it out. Unfortunately I'm not able to authenticate into LastPass. I'm wondering if it has something to do with my extremely long master password. It's over 128 characters and it's so long that it's not able to fit inside of the input box which makes it seem like the input box is getting cleared out. Is that possible?

I've confirmed that I can still auth into LP via the website so I definitely know my master password :)

**Comment by ajrosen** [comment]
Thanks for trying it!

I changed my password to something over 128 characters and was still able to login with the workflow.  Does your have only printable ASCII characters, or are there Unicode characters?

Are you able to login from the command line?
```
export LPASS_DISABLE_PINENTRY=1
echo "<super long password here>" | lpass login "<your username here>"
```

**Comment by johnhaley81** [comment]
Ok so I was able to auth via the CLI but it was kind of awkward. When prompted for my 2FA I typed in the code and it just hung for a few minutes. Then after I figured it just timed-out I tried to `CTRL-C` out of it which at that point it asked me for my code again. I figured maybe something got borked so I typed it in again and it immediately logged me in.

So maybe it's something with how the 2FA code is being passed? Possibly a bug with `lastpass-cli`.

Anyways I'm able to log in now and the workflow is working great!
